src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Figure implements Information, Area {
+    private Color color;
+
+    @Override
+    public double area() {
+        return 0;
+    }
+
+    @Override
+    public void printInformation() {
+    }
+
+    public Color getColor() {
+        return color;
+    }
+
+    public void setColor(Color color) {
+        this.color = color;
+    }
+
+}
```

Comment:
Figure looks more like `abstract` class. In that case, you don't need to override abstract methods and give them default realisation.